### ISSUE TITLE
Regarding the csv in the repo

### ISSUE THREAD

**Issue by shubhaminnani** [ls1rius/WSI_FiVE#7]
Hi,

Thank you for the amazing work. I was trying to implement the same code on different dataset.
I already have features extracted from WSI images and I have the text prompts from GPT in the csvs. 

I notice that there are few files in the config, how are these files created? Are this required for pre-computed features? 

Can you help to understand what this below files indicate?

LUAD_LUSC_data_desc_reid.csv
LUAD_LUSC_data_desc_reid_delval.csv 
LUAD_LUSC_data_train_reid.csv 
LUAD_LUSC_data_val_reid.csv
LUAD_LUSC_labels_desc_reid.csv
LUAD_LUSC_labels_reid.csv 
LUAD_LUSC_labels_train_reid.csv
LUAD_LUSC_labels_val_reid.csv

Thanks,
Shubham

**Comment by ls1rius** [comment]
Thank you for your interest in our work.
Here are some explanations of the files:

LUAD_LUSC_data_desc_reid.csv (data for training, report and wsi task, you dont need to use this)
LUAD_LUSC_data_desc_reid_delval.csv (data for training, report and wsi task, delete the parts that overlap with the validation dataset)
LUAD_LUSC_data_train_reid.csv (data for training, binary classification task, finetune)
LUAD_LUSC_data_val_reid.csv (data for validation, binary classification task, finetune)
LUAD_LUSC_labels_desc_reid.csv (label for training, report and wsi task)
LUAD_LUSC_labels_reid.csv (useless)
LUAD_LUSC_labels_train_reid.csv  (label for training, binary classification task, finetune)
LUAD_LUSC_labels_val_reid.csv (label for validation ,binary classification task, finetune)

LUAD_LUSC_data_desc_reid.csv contains all the data. In order to remove the hidden influence of information leakage, even though we don't use any binary classification information, we deleted the data of the validation set from it. So we get the LUAD_LUSC_data_desc_reid_delval.csv.

If you have any other questions, please feel free to contact me.
